src/explore-education-statistics-admin/src/pages/release/contexts/ReleaseContext.tsx

```diff
@@ -1,43 +1,45 @@
-import { Release } from '@admin/services/releaseService';
+import { ReleaseVersion } from '@admin/services/releaseService';
 import noop from 'lodash/noop';
 import React, { createContext, ReactNode, useContext, useMemo } from 'react';
 
-export interface ReleaseContextState {
-  release: Release;
-  releaseId: string;
+export interface ReleaseVersionContextState {
```

Comment:
I'm not sure that this and the other exports in this file need to be renamed. If they do then can the file be renamed as well to be consistent? (I'd be looking for a file called `ReleaseVersionContext` if I saw `ReleaseVersionContextProvider`).